db/migrations/00_create_database.sql

```diff
@@ -0,0 +1,2 @@
+CREATE DATABASE makersbnb ;
```

Comment:
Do not need this file. can just be a note in the readme

Reply:
Done